ctags.lua

```diff
@@ -6,6 +6,8 @@ local tags = {'tags'}
 local function get_path(prefix, path)
 	if string.find(path, '^./') ~= nil then
 		path = path:sub(3)
+	elseif string.find(path, '^/') ~= nil then
```

Comment:
This could be a bit easier to follow if we performed this check first and then returned absolute paths early.